src/Panel/Right.php

```diff
@@ -152,10 +152,12 @@ public function addConfirmation(string $msg, string $title = 'Closing panel!', s
     /**
      * Callback to execute when panel open if dynamic content is set.
      * Differ the callback execution to the FlyoutContent.
+     *
+     * @param \Closure(object): void $fx
```

Comment:
TODO `object` seems to pass phpstan, but recheck if better type can be used before merge